spec/preload.lua

```diff
@@ -1,12 +1,15 @@
 -- This script is given to Busted via the --helper argument. Modules loaded here
 -- won't be cleared and reloaded by Busted. This is needed for lgi because lgi
 -- is not safe to reload and yet Busted manages to do this.
-require("lgi")
+require "lgi"
 
 -- Always show deprecated messages
-_G.awesome = {version = "v9999"}
+_G.awesome = {
+   version = "v9999",
+   api_level = 9999,
+}
 
 -- "fix" some intentional beautiful breakage done by .travis.yml
-require("beautiful").init{a_key="a_value"}
+require("beautiful").init { a_key = "a_value" }
```

Comment:
this formatting change in a totally random place and this space in `init {` doesnt match the rest of codebase